server/graphql-engine.cabal

```diff
@@ -194,6 +194,9 @@ library
                      
                      , directory
 
+                     -- regex
+                     , regex-posix
```

Comment:
We already depend on `regex-tdfa`, so can we use that instead of adding another dependency?

Reply:
@lexi-lambda I have updated the code to use `regex-tdfa`